### PR TITLE
New Health Check report for mixed account types

### LINKED ISSUE #2130: Correct Health Check reports so they accurately pull up results
This https://powerofus.force.com/0D58000002mCg9G is coming up quite a bit. We should have made this report using the Accounts report type, not the Contacts & Accounts report type. There are Accounts that are properly linked to the correct Primary Contact so they're not coming up in the Health Check sweep for Accounts without Contacts, but their Contacts don't have the Account as their direct Account so they're not coming up in the Account Type report.

Recommendations:

* Add a report for 'Accounts by AccountType' using Accounts report type with same _SYSTEM:AccountType not equal to (blank) filter and reference that in the error about mix of account types
* Add a report for 'Individual Accounts without Linked Contact' using the Accounts report type with filters of _SYSTEM:AccountType not equal to (blank) and Primary Contact equals (blank) and reference that in the error about orphaned Accounts, which is what it's really looking for.

### PR DESCRIPTION
**Pull request by dospromptman**
Adding new report to better show which accounts are violating the mixed
account model.  Update health check messaging to direct users to the new
report.

# Info
* "Individual Accounts by Account Type" is a new report that lets you view all Accounts, aggregated by their Account type.  This report is recommended by Health Check for diagnosing issues related to organizations that are using a mixed Account model.

# Issues
Fixes #2130

### CODE REVIEW

**Comment by dospromptman** [comment]
@jvarese, I have assigned you as reviewer to approve two changes/additional strings.

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -2028,7 +2028,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>healthSolutionAccountModel</shortDescription>
-        <value>All of your individual Contacts should use the same Account Model. View these Contacts by running the &lt;b&gt;Individual Contacts by Account Type&lt;/b&gt; report in the NPSP Health Check reports folder.</value>
+        <value>All of your Accounts should use the same Account Model. View these Accounts by running the &lt;b&gt;Individual Accounts by Account Type&lt;/b&gt; report in the NPSP Health Check reports folder.</value>
```
Comment (dospromptman) [inline src/labels/CustomLabels.labels]: @jvarese The solution message text for the Account Model Data health check has been updated to refer to the new report.

screen shot 2016-07-18 at 8 12 44 pm

File: src/reports/NPSP_Health_Check/Individual_Accounts_by_Account_Type.report [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <columns>
+        <field>DUE_DATE</field>
+    </columns>
+    <columns>
+        <field>LAST_UPDATE</field>
+    </columns>
+    <columns>
+        <field>ADDRESS1_STATE</field>
+    </columns>
+    <description>Lists Accounts grouped by their system Account Type.  Details are hidden to avoid data limits.  Check the group you want to see details on and click the Drill Down button.</description>
```
Comment (dospromptman) [inline src/reports/NPSP_Health_Check/Individual_Accounts_by_Account_Type.report]: @jvarese I copied over the description from the old Individual Contacts by Account Type health check report.  This message is visible in the report selection list, in the Reports tab.

screen shot 2016-07-18 at 8 13 11 pm

**Comment by judisohn** [comment]
@dospromptman Wow, thanks for getting to this so quickly after our conversation yesterday!

**Comment by davidhabib** [comment]
I'm ready to merge once @jvarese ok's the test strings.

File: src/labels/CustomLabels.labels (hunk 1 above)
Comment (jvarese) [inline src/labels/CustomLabels.labels]: @dospromptman @davidhabib 

The Account Model is set to Household Account, but Health Check has found that you're using a mixture of Account types.

All of your Accounts should use the same Account Model. View all of your Accounts and their types by running the **Individual Accounts by Account Type** report in the NPSP Health Check reports folder.

File: src/reports/NPSP_Health_Check/Individual_Accounts_by_Account_Type.report (hunk 2 above)
Comment (jvarese) [inline src/reports/NPSP_Health_Check/Individual_Accounts_by_Account_Type.report]: @dospromptman @davidhabib 

Thumbs up.

File: src/labels/CustomLabels.labels (hunk 1 above)
Comment (davidhabib) [inline src/labels/CustomLabels.labels]: @dospromptman let me know when you've updated the strings per Jon's feedback. thanks.

**Comment by dospromptman** [comment]
Ok, I have updated the labels based on @jvarese's feedback.  Ready to merge, @davidhabib.

**Pull request merged**